src/store/mutations.js

```diff
@@ -108,7 +108,9 @@ export function setCurrentConditions(state, condition) {
     if (index === -1) {
         state.currentConditions.push(condition);
     } else {
-        Vue.set(state.currentConditions, index, condition);
+        let conditionFixed = condition;
```

Comment:
The problem is that we process here all conditions and they have different structure. ENUM and STATUS have operator on 2nd place, but others not. It breaks all other cases. You can try to add some numeric condition or zygosity.